scripts/dev/generate_epJSON_schema/idd_parser.py

```diff
@@ -111,14 +112,17 @@ def parse_idd(data):
             eat_comment(data)
         elif token == TOKEN_GROUP:
             next_token(data)
-            eat_comment(data)
+            current_group_name = parse_line(data)
```

Comment:
I figured I could leverage the existing ones, but I wasn't immediately sure.  This is great.